src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,76 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String LINE_SEPARATOR = System.lineSeparator();
+
     public void getStatistic(String fromFileName, String toFileName) {
+        String[][] data = readData(fromFileName);
+
+        String resultData = calculateStatistics(data);
+
+        writeData(toFileName, resultData);
+    }
+
+    public String[][] readData(String fileName) {
+        //Read from file
```

Comment:
There is an unnecessary blank line at the beginning of the `readData` method body. It's good practice to remove extra whitespace for better code readability.